### ISSUE TITLE
Logger unexpectedly merging objects

### ISSUE THREAD

**Issue by tony-g** [logdna/logdna-winston#12]
Hi, trying out logdna, and with the following:

    "logdna-winston": "2.2.2",
    "winston": "3.2.1"

```js
  const options = {
    key: apiKey,
    app,
    env,
    index_meta: true
  }

  const winstLogger = winston.createLogger({
    transports: [
      new logdnaWinston(options),
      new winston.transports.Console({
        format: winston.format.simple()
      })
    ]
  })

  winstLogger.info('1', { '1': true, x: { a: 1 } })
  winstLogger.info('2', { '2': true, x: { b: 2 } })
```

I'd expect the 2 lines that get written to the console:
```
info: 1 {"1":true,"x":{"a":1}}
info: 2 {"2":true,"x":{"b":2}}
```

In practice, I'm getting 2 identical lines:
```json
{
  "1": true,
  "2": true,
  "x": {
    "b": 2
  }
}
```

![image](https://user-images.githubusercontent.com/834042/63650587-69530080-c74c-11e9-8308-958afceb11ae.png)

I'm not mutating any of the logged objects and didn't see anything in the docs about auto-merging, so I'm a bit confused.  This also seems to work fine with other services.

Thanks!

**Comment by elsbree** [comment]
We're seeing this as well- we log many lines per second, and it's seriously impacting our ability to filter logs effectively.

**Comment by elsbree** [comment]
I've submitted a [PR](https://github.com/logdna/nodejs/pull/70) to fix this, but in the meantime reverting to 2.2.0 will resolve the problem.
